axelrod/strategies/axelrod_second.py

```diff
@@ -975,3 +975,294 @@ def strategy(self, opponent: Player) -> Action:
                 self.defect_padding = 0
 
             return self.try_return(opponent.history[-1])
+
+
+class Harrington(Player):
+    """
+    Strategy submitted to Axelrod's second tournament by Paul Harringtn (K75R),
+    and came in eighth in that tournament.
+
+    This strategy has three modes:  Normal, and Fair-weather, Defect.
```

Comment:
`Normal, Fair-weather and Defect`

Reply:
Looking at the source code https://github.com/Axelrod-Python/TourExec/blob/master/src/strategies/K75R.f, these names are of your choosing? Nothing wrong with that. Perhaps, we can add a sentence saying that though (just to avoid potential confusion):

```
..., and Defect. These mode names were not present in Harrington's submission.
```

Reply:
Could we also add a sentence along the lines of what you wrote in the pdf: that the `Fair-weather` mode occurs only in 1 very specific type of situation.